docs/source/install/kernel_install.rst

```diff
@@ -0,0 +1,16 @@
+.. _kernel_install:
+
+Kernel Installation
+-------------------
+
+IPython can be installed (different python versions, virtualenv or conda environments) can
+be installed as a kernel by following these steps:
+
+* make sure that the desired python installation is active (e.g. activate the environment)
+  and ipython is installed
+* run once ``ipython kernelspec install-self --user`` (or ``ipython2 ...`` or ``ipython3 ...``
+  if you want to install specific python versions)
+
+The last command installs a :ref:`kernel spec <jupyter-client:kernel-specs>` file for the current python installation in
+``~/.jupyter/kernels/``. Kernel spec files are JSON files, which can be viewed and changed with a
```

Comment:
This isn't quite right - the exact location of the kernel specs now depends on the platform. Probably best to leave out the mention of the path.